src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1948,7 +1949,13 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                         }
                         isVisibleResponse = false;
                     }
-
+                }
+                
+                if (question.questionType == FeedbackQuestionType.CONTRIB) {
```

Comment:
Hm, to do so I would need some way to make sure that the team names are for teams in the section that the instructor is allowed to view. That would probably mean that I still need to get the instructor from the database to check if he is allowed for the privilege :/